### ISSUE TITLE
Need way to identify conversations that have customized notifications

### ISSUE THREAD

**Issue by Bill-Stewart** [moezbhatti/qksms#1495]
Hi Moez -- it would be very helpful if there was a way to identify conversations that have customized notifications. Thanks!

**Comment by moezbhatti** [comment]
I agree. I think there also needs to be a simple way to reset per-conversation settings. This issue is separate from that one

**Comment by TheShanMan** [comment]
> I agree. I think there also needs to be a simple way to reset per-conversation settings. This issue is separate from that one

Is there _any_ way to reset per-conversation settings? And does changing the conversation color force the notification sound to be de-coupled from the global notification sound? I think that's what happened to me but can't say for sure. I'd just like to get some conversation sounds re-coupled to the global setting somehow if it's possible.
